parl/remote/utils.py

```diff
@@ -94,3 +103,28 @@ def redirect_stdout_to_file(file_path):
     finally:
         sys.stdout = tmp
         f.close()
+
+
+def locate_remote_file(module_path):
```

Comment:
Maybe we can give some examples of this function.

Reply:
done